EO + projects

airon90 opened the pull request:
This is my very first commit.
I corrected EO translations and added some projetcs and licenses.
The problem is that I don't have any image to be added and I don't know how to work with pictures.

Commit:
Update js/locales/eo.json

Commit:
+mailoo +ethercalc +cpal +various license

Commit:
+pixlpaste

Commit:
reorder +projects

bit.ly, tinyurl and picasa as alternatives to
pixlpaste, ur1.ca, jslander and Project Khalio as projects

jancborchardt commented on js/data.json:
```diff
@@ -49,15 +54,22 @@ var lp = {
 		{"id":"freenode","category":"social","name":"Freenode","description":"IRC","address":"http://webchat.freenode.net/","licenses":["gnu-gpl-v2"],"tags":["irc","chat"],"introduction":"Freenode provides discussion facilities for the Free and Open Source Software communities, for not-for-profit organizations and for related communities and organizations.","source":"https://hg.qwebirc.org/qwebirc"},
+		{"id":"friendica","category":"social","name":"Friendica","description":"social multi network","address":"http://friendica.com/","licenses":["mit"],"tags":["social","network","decentralized","microblog"],"alternative":["facebook","google plus","twitter"],"introduction":"Friendica is is an open source, free social web server. Your 'friends' can be from Facebook, Diaspora, Twitter, Identi.ca, webl ...
+		{"id":"mailoo","category":"social","name":"Mailoo","description":"mail service","address":"http://mailoo.org/","licenses":["various"],"tags":["mail"],"alternative":["gmail", "yahoo", "hotmail"],"introduction": "Mail service based on free software"},
+		{"id":"ur1.ca","category":"social","name":"ur1.ca","description":"Free URL shortening","address":"http://ur1.ca/","licenses":["gnu-gpl-v3"],"tags":["sharing","url"],"introduction": "Free URL shortening powered by StatusNet Inc.","alternative":["google","bitly","tinyurl"],"source":"http://ur1.ca/ur1-source.tar.gz"},
```
I’d say ur1.ca would be »tools« rather than »social«. What do you think? cc @jhuet

jancborchardt commented on js/data.json:
```diff
@@ -49,15 +54,22 @@ var lp = {
+
 		{"id":"seeks","category":"tools","name":"Seeks","description":"search engine","address":"http://seeks.fr/","licenses":["gnu-agpl-v3"],"tags":["searchengine"],"alternative":["google","bing","yahoo"],"introduction":"Seeks is a free and open technical design and application for enabling social websearch. Its specific purpose is to regroup users whose queries are similar so they can share both the  ...
 		{"id":"etherpad","category":"tools","name":"Titanpad","description":"document editor","address":"http://titanpad.com/","licenses":["apache-v2"],"tags":["collaborative","document","editor"],"introduction":"Web-based realtime collaborative document editor.","source":"https://github.com/titanpad/titanpad"},
+		{"id":"ethercalc","category":"tools","name":"Ethercalc","description":"spreadsheet editor","address":"http://ethercalc.net/","licenses":["cpal-v1"],"tags":["collaborative","stitanpadpreadsheet","editor"],"introduction":"Web-based realtime collaborative spreadsheet editor.","source":"https://github.com/audreyt/ethercalc"},
```
The tag »stitanpadpreadsheet« seems weird, is it supposed to be there?

jancborchardt commented:
Whoa, looks really good! I just added 2 comments inline, about the category of ur1.ca and a tag of EtherCalc.

@jhuet can you check it out too? And do we have anyone who can review the Esperanto changes, @sindikat maybe?

airon90 commented:
@jancborchardt I wrote "spreadsheet" but "titanpad" has been added just before the first letter. I don't know why, my PC sucks after the installation of Arch Linux. :/
You're right for the category of ur1.ca too. I thought that its usage is perfect for social network so I added "social".
I don't know who created the Esperanto translation but (s)he made many big mistakes e.g. (s)he often forgot the accusative form or s(he) added it only to nouns but not to adjectives.
During these days I will add more projects and add more info to the existent ones. Could someone tell me how to update my fork after the pull?

Commit:
-openphoto

https://github.com/libreprojects/libreprojects/issues/30

Commit:
+wikivojage +sources +fix

Wikivojage is part of Wikimedia Foundation, Wikitravel has been bought by Internet Brands Inc.
Added sources to WMF projects (dump site)
Fixed some errors

airon90 commented:
I'm closing this pull request because another one will be opened soon with more adds
